packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.ts

```diff
@@ -782,43 +789,39 @@ export const InstanceProperties = {
   description: define(null, ["es.symbol", "es.symbol.description"]),
   dotAll: define(null, ["es.regexp.dot-all"]),
   drop: define(null, [
+    "es.iterator.drop",
+    ...IteratorDependencies,
     "esnext.async-iterator.drop",
     ...AsyncIteratorDependencies,
-    "esnext.iterator.drop",
-    ...IteratorDependencies,
-  ]),
-  emplace: define("instance/emplace", [
-    "esnext.map.emplace",
-    "esnext.weak-map.emplace",
   ]),
   endsWith: define("instance/ends-with", ["es.string.ends-with"]),
   entries: define("instance/entries", ArrayNatureIteratorsWithTag),
   every: define("instance/every", [
     "es.array.every",
-    "esnext.async-iterator.every",
```

Comment:
Because lack of dependencies, it could cause a error, so this and similar cases commented. They are covered by `AsyncIteratorProblemMethods`.